### ISSUE TITLE
use of undeclared type "TDBadgedCell"

### ISSUE THREAD

**Issue by BendingUnit-22** [tmdvs/TDBadgedCell#74]
# Using TDBadgedCell (4.1.1)
swift 3
I did use import TDBadgedCell if you were wondering

**Comment by codeSupreme** [comment]
@BendingUnit-22  how did you fix this?  I'm having the same issue.

**Comment by tmdvs** [comment]
If you're using Swift you shouldn't need to import the class as it should be automatically available to you (see the example project).

Are you trying to bridge the Swift 3 version into an Objective-C project?

**Comment by codeSupreme** [comment]
I'm using Swift via a cocoa pod.  But I'm trying to subclass it which is
giving me the error.

**Comment by tmdvs** [comment]
I added the following test subclass to the demo project and the project compiled without any issues. Can you confirm that CocoaPods is adding the class correctly? Are you able to use TDBagdedCell itself?

```swift
//  TestCellClass.swift

import UIKit

class TestCellClass: TDBadgedCell {

}
```

**Comment by codeSupreme** [comment]
Yes, the demo project would work fine because the TDBadgedCell.swift is in
the project but once you use cocoapods to install the TDBadgedCell you will
error out when trying to subclass.

**Comment by codingalex** [comment]
I have excactly the same issue: I use the Swift version via Cocoapods and subclassing is not possible.

**Comment by tmdvs** [comment]
This sounds more like an issue with CocoaPods?

**Comment by tmdvs** [comment]
I've just released an update (2a0be57) which changes the control type to open. I believe this should expose the class for subclassing.

I'm also updating the pod, so feel free to run pod update.
